src/_micro-api/libraries/signalDecoder/src/signalDecoder.cpp

```diff
@@ -934,6 +862,14 @@ void SignalDetectorClass::processMessage()
 }
 
 
+/* function to convert to HEX without a leading zero */
+void SignalDetectorClass::SDC_PRINTtoHEX(unsigned int numberToPrint) {  // smaller memory variant for sprintf hex output ( sprintf(buf, "R%X;", value) )
```

Comment:
Bei dem Namen der Funktion bin ich am überlegen, was da zu Hex geht. Eigentlich ist es ja ein int.
SDC_PRINT_Hex wäre aus meiner Sicht passender, denn eigentlich wird ja der int in ein hex und anschließend ausgegeben.

Reply:
Mach einen Namensvorschlag und wir benennen diese um.
Vorschlag `SDC_PRINT_intToHex `